modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -123,14 +123,11 @@ vec4 project_position(vec4 position, vec3 position64Low) {
       );
     }
   }
-  if (project_uProjectionMode == PROJECTION_MODE_WEB_MERCATOR_AUTO_OFFSET &&
+  if (project_uProjectionMode == PROJECTION_MODE_IDENTITY ||
+    project_uProjectionMode == PROJECTION_MODE_WEB_MERCATOR_AUTO_OFFSET &&
```

Comment:
Put some extra parentheses here to clarify which conditions are grouped together.